.dockerignore

```diff
@@ -6,7 +6,7 @@
 __pycache__/
 
 # project
-*/saved/
+./saved/
```

Comment:
I want ignore `./lmnet/saved/` that's why `*/saved/`.

Reply:
@ruimashita 
Oh ok, but currently nobody use `./lmnet/saved/` . right?
Or, should I write both `./saved/` and `*/saved/`?

Reply:
Thank you! @iizukak 
> Or, should I write both ./saved/ and */saved/?

Writing both is LGTM.
But it is better to use `saved/` instead of `./saved/`

Reply:
OK, I'll write both `saved/` and `*/saved/`.